DCO

```diff
@@ -2,8 +2,9 @@ Developer Certificate of Origin
 Version 1.1
 
 Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
-660 York Street, Suite 102,
-San Francisco, CA 94110 USA
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
```

Comment:
No comma is needed between the state and the ZIP

Reply:
> No comma is needed between the state and the ZIP

This is what came out of [`curl`ing https://developercertificate.org][1].  With the DCO distributed under [verbatim-copies-only][2] terms, I don't think it's worth tuning their content.  Maybe submit these suggestions upstream to the LF?

[1]: https://github.com/openshift/installer/pull/26#issue-199841667
[2]: https://github.com/openshift/installer/blob/9811c32af5bffebf42237cb94068f008655a307c/DCO#L8-L9

Reply:
sorry I missed that. this in indeed the content of the hosted document :/